src/server/Debug.server.lua

```diff
@@ -0,0 +1,7 @@
+local ReplicatedStorage = game:GetService("ReplicatedStorage")
+local Packages = ReplicatedStorage:WaitForChild("Packages")
+local Link = require(Packages:WaitForChild("link"))
+local GetPing = Link:CreateFunction("GetPing")
+GetPing:OnServerInvoke(function()
+	return
```

Comment:
should return an actual value

Reply:
what for? i've tried doing that and i get nothing back.